File: resources/kotlin/loops.kt
fun main() {
    val collection = arrayOf("this", "is", "a", "collection")

    for (i in 1..3) {
        for (item in collection) {
            println(item)
        }
    }

    var x: Int = 10
    var z: Int = 11

    while (x > 0) {
        x--
    }

    do {
        val y = z % 7
        z++
    } while (y != 0)
}

File: resources/python/loops.py
list = ['One', 'Two', 'Three']

# For-loop to iterate over the list:
for item in list:
    # Nested loop to iterate over characters:
    for chara in item:
        print(chara)

for item in list:
    print(item)
else:
    print('hello')

while True:
    for i in range(10):
        print(i)
    break

num = 0
while num < 3:
    num+=1
else:
    print("end")
